.github/scripts/createAppVersionPr.ts

```diff
@@ -0,0 +1,34 @@
+const { echo, exec, ShellString, env, config } = require('shelljs')
+
+config.fatal = true
+
+const branchName = new ShellString(env.BRANCH_NAME)
+
+// ensure that we are using ssh
+exec('git remote set-url origin git@github.com:valora-inc/wallet.git')
+
+echo('Create version bump branch from main')
+exec(`git checkout -b ${branchName}`)
+
+echo('Bump app version')
+exec('yarn pre-deploy --minor')
+
+const appVersion = exec('node -p "require(\'./package.json\').version"')
+
+echo('Push changes to branch')
+exec('git add .')
+exec('git commit -m "Bump app version to $app_version"')
+exec(`git push --set-upstream origin ${branchName}`)
+
+echo('Open version bump PR')
+exec(
+  `
+  curl -u "valora-bot:$VALORA_BOT_TOKEN" \
+  -X POST \
+  -H "Accept: application/vnd.github.v3+json" \
+  https://api.github.com/repos/valora-inc/wallet/pulls \
+  -d '{ "head": "'${branchName}'", "base": "main", "title": "Bump app version to '${appVersion}'" }'
+`
+)
+
+export {}
```

Comment:
Can we remove this?

Reply:
no :( the lint fails because typescript complains that `Cannot redeclare block-scoped variable` even though the variables are in different files. this was my hack to get around the scope, do you have a better idea??

Reply:
This converts it to an ES module so variables are scoped locally right? Could we just change require's to imports instead of this export stmt? Also would be good to make similar changes in `createLicenseDisclaimerPr.ts`

Reply:
ah idk why i missed the import vs require thing (i cannot remember why i used require in the createLicenseDisclaimerPr script to begin with, probably because it started off as a .js file and then i forgot about it...) - i've updated to use import and everything is fine now :) also made a couple of small changes to make the script work better. thanks for the feedback!!